Makefile

```diff
@@ -348,16 +348,13 @@ test-unit-cover: ARGS=-timeout=15m -race -coverprofile=coverage.txt -covermode=a
 test-unit-cover: TEST_PACKAGES=$(PACKAGES_UNIT)
 
 test-upgrade:
-ifeq (, $(TARGET_VERSION))
-	@make build-docker
-endif
```

Comment:
I believe we need this for the update tests? Or is the "new" desired workflow, that it's mandatory to manually to build the docker image before? I thought it was quite handy that it does this automatically - it was just not needed for ALL tests if my understanding is right

Reply:
after testing once more I realised this actually worked, don't why I saw a failure first time around..